webapp/src/components/preview-movies/fetch-movies/MoviesByGenre.tsx

```diff
@@ -1,17 +1,17 @@
 import React, { useEffect, useState } from 'react';
 import { getGlobalInstance } from 'plume-ts-di';
 import useLoader from '../../../lib/plume-http-react-hook-loader/promiseLoaderHook';
-import PreviewMoviesService from '../../../services/preview-movies/PreviewMoviesService';
 import { Production } from '../../../api/types/MovieDbTypes';
 import Row from '../style/row/Row';
+import MovieService from '../../../services/movie/MovieService';
 
 function MoviesByGenre({ genre, id_genre }:{ genre:string, id_genre: number }) {
```

Comment:
camelCase et non snake_case.

Snake_case c'est plus en python.

```suggestion
function MoviesByGenre({ genre, idGenre }:{ genre:string, idGenre: number }) {
```